bin/templates/cordova/lib/builders/ProjectBuilder.js

```diff
@@ -17,11 +17,10 @@
        under the License.
 */
 
-var Q = require('q');
```

Comment:
I believe this should remain in this PR and would be removed in : https://github.com/apache/cordova-android/pull/833

Reply:
Module imports are removed _iff_ the module is not used. Otherwise ESLint would complain.